### DISCUSSION TITLE
Tutorial Examples

### DISCUSSION THREAD

**Discussion by GeoCalcul** [Anura3D/Anura3D_OpenSource#170] [General]
Hello 
I have modelled 2 tutorial examples 1) Submerge slope collapse and 2) Slope failure under transient hydraulic loads.
I did not have any errors in modelling and results. However, my results do not match with the manuals and it does not show any deformation.
Regards,
Al

**Comment by murphy-john** [comment]
Could you show more? It's hard to say what didn't work. I'd recommend checking the calculation parameters. 

Two possibilities are gravity might not be turned "on" or the time per load step is too small.

**Reply by GeoCalcul** [reply]
Thank you for your reply.
It was because of gravity. However, I noticed when you make any changes in GiD, you should generate mesh again. otherwise, the GOM file will not be updated.

**Reply by murphy-john** [reply]
Yes, you are correct. There is a warning message when creating your model in GiD to re-generate your mesh every time you make a change to any of the model parameters.
